### ISSUE TITLE
API Version and Schemes

### ISSUE THREAD

**Issue by mdigger** [raphael/goa#138]
I create a test description:
```go
package design

import (
	. "github.com/raphael/goa/design"
	. "github.com/raphael/goa/design/dsl"
)

var _ = API("I_love_goa", func() {
	Title("Test of API")
	Host("localhost:8080")
	Scheme("http")
	BasePath("/api/v1")
})

var _ = Resource("test", func() {
	BasePath("/test")
	DefaultMedia(TestMedia)
	Action("list", func() {
		Routing(
			GET(""),
		)
		Response(OK)
	})
})

var TestMedia = MediaType("application/vnd.test+json", func() {
	Attributes(func() {
		Attribute("id", String, "ID of object")
	})
	View("default", func() {
		Attribute("id")
		Required("id")
	})
})
```
 and generate `swagger.json` file:
```json
{  
   "swagger":"2.0",
   "info":{  
      "title":"Test of API",
      "version":""
   },
   "host":"localhost:8080",
   "basePath":"/api/v1",
   "schemes":[  
      "http"
   ],
   "consumes":[  
      "application/json"
   ],
   "produces":[  
      "application/json"
   ],
   "paths":{  
      "/test":{  
         "get":{  
            "operationId":"test#list",
            "consumes":[  
               "application/json"
            ],
            "produces":[  
               "application/json"
            ],
            "responses":{  
               "200":{  
                  "description":"",
                  "schema":{  
                     "$ref":"#/definitions/Test"
                  }
               }
            },
            "schemes":[  
               "https"
            ]
         }
      }
   },
   "definitions":{  
      "Test":{  
         "title":"Mediatype identifier: application/vnd.test+json",
         "type":"object",
         "properties":{  
            "id":{  
               "type":"string",
               "description":"ID of place"
            }
         }
      }
   }
}
```

- How I can set a `info.version` in definition?
- In `schemes` defined only `http`, but in resource definition I have `https`.
- Is `consumes` and `produces` always only `application/json`?
- Empty `description` in `responses`.

**Comment by mdigger** [comment]
OK! I found all the answers in the source code. :)
Thanks!

**Comment by raphael** [comment]
Thanks for the report, you found the answers but I don't think they were all satisfactory ;)
* API versioning is work under way in the `versioning_support` branch. See the [wiki](https://github.com/raphael/goa/wiki/goa-Roadmap).
* I fixed the bug with operation schemes always being `https`. It is now also possible to specify an action specific scheme in the DSL.
* For now `consumes` and `produces` are always JSON. It would be nice for goa to support other encodings in the future though.
* Fixed the empty description in #142

**Comment by robbertdesmit** [comment]
hi  @raphael  i i can't see the wiki anymore :-) is there any info about the versioning_support ?
